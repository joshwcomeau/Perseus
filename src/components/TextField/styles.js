import { StyleSheet } from '../../helpers/global-aphrodite';

export default StyleSheet.create({
  textField: {

  },
});
