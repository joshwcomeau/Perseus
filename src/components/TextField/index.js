// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import { css } from '../../helpers/global-aphrodite';

import styles from './styles';


const TextField = () => {
  return (
    <div className={css(styles.textField)}>
      Your Component Here :)
    </div>
  );
};

TextField.propTypes = {

};

TextField.defaultProps = {

};

export default TextField;
