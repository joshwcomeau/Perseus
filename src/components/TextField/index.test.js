/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import TextField from './index';

describe('TextField', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<TextField />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
