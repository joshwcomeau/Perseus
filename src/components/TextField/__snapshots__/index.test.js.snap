exports[`TextField renders correctly 1`] = `
<div
  className="textField_120drhm">
  Your Component Here :)
</div>
`;
